corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -275,6 +275,19 @@ define("cloudcare/js/form_entry/entries", [
             return null;
         };
 
+        self.afterRender = function (elements) {
+            _.each(elements, function (el) {
+                if (el.nodeName === 'TEXTAREA') {
+                    const showFullText = () => {
+                        el.style.height = 'auto';
+                        el.style.height = el.scrollHeight + 3 + 'px';
```

Comment:
How come `el.style.height` needs to be set twice?

Reply:
I'm not 100% sure on this, but I think there's an interdependency between `height` and `scrollHeight`, so the `auto` resets the former.  Without that line there, the 3px gets added every character you type, and it won't shrink back down if you remove text (that's from memory, I might be off-base).